src/overcooked_demo/server/static/js/tutorial.js

```diff
@@ -0,0 +1,285 @@
+// Persistent network connection that will be used to transmit real-time data
```

Comment:
Same here

Reply:
This one is technically not in use, though based on the code this setup supports human vs. human game mode, though the setup is more complicated. This is required if the game is setup on the server,  though it is not required for AI vs. AI games or AI vs Human games.